staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics.go

```diff
@@ -39,6 +39,15 @@ var (
 		[]string{"resource"},
 	)
 
+	eventsCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "apiserver_events_total",
+			Help:           "Counter of events proceeded in watchcache broken by resource type.",
```

Comment:
proceeded - processed or dispatched?

Reply:
Switched to dispatched (I don't have strong preference).